### PR TITLE
same tab names for different indexes

### PR DESCRIPTION
**Pull request by newbfg**

### CODE REVIEW

File: catboost/python-package/catboost/widget/ipython/accurancy/catboost_training.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
```
Comment (annaveronika) [inline catboost/python-package/catboost/widget/ipython/accurancy/catboost_training.json]: ipython => examples

File: catboost/python-package/catboost/widget/ipython/accurancy/catboost_training.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+"meta":{"launch_mode":"Train","name":"experiment","iteration_count":50,"learn_metrics":[{"best_value":"Min","name":"Logloss"},{"best_value":"Max","name":"Accuracy"}],"test_sets":["test"],"test_metrics":[{"best_value":"Min","name":"Logloss"},{"best_value":"Max","name":"AUC"},{"best_value":"Max","name":"Accuracy"}],"learn_sets":["learn"]},
```
Comment (annaveronika) [inline catboost/python-package/catboost/widget/ipython/accurancy/catboost_training.json]: accuraNcy => accuracy

File: catboost/python-package/catboost/widget/ipython/accurancy/catboost_training.json (hunk 1 above)
Comment (newbfg) [reply catboost/python-package/catboost/widget/ipython/accurancy/catboost_training.json]: It's all about ipython

File: catboost/python-package/catboost/widget/ipython/accurancy/catboost_training.json (hunk 2 above)
Comment (newbfg) [reply catboost/python-package/catboost/widget/ipython/accurancy/catboost_training.json]: done

**Comment by annaveronika** [comment]
@arcadia-devtools Ship it!

**Pull request merged**